app/views/article/upload_form.html.slim

```diff
@@ -0,0 +1,16 @@
+=javascript_include_tag 'application'
+.litebox-embed(style="width:550px")
+  h1 Upload Subjects
+  p To create subjects from an external authority file, upload a spreadsheet with one row per subject. See #{link_to "link", article_upload_example_path, {target: "_blank"}} for an example. (Note that multi-line/article cannot be updated.)
```

Comment:
You didn't include upload_example/subject_example.csv in this PR.

Reply:
Does the spreadsheet support comments?  If it does, can we move this to it?:
"(Note that multi-line/article cannot be updated.)" 
If not, we should document it in documentation.  I wouldn't include it here.

Reply:
Instead of "See #{link_to "link", article_upload_example_path, {target: "_blank"}} for an example."  Just link the "spreadsheet with one row per subject" words.  That will mirror what we did for metadata.yml.

Reply:
Looks like our `.gitignore` prevented me from adding the example csv.